targets/phase4/main.cpp

```diff
@@ -89,43 +62,26 @@ int main() {
     // Attempt to join the CAN network
 
     ///////////////////////////////////////////////////////////////////////////
-    // Setup CAN configuration, this handles making drivers, applying settings.
+    // Setup CANOpen, this handles making drivers, applying settings.
     // And generally creating the CANopen stack node which is the interface
     // between the application (the code we write) and the physical CAN network
     ///////////////////////////////////////////////////////////////////////////
     // Make drivers
 
-    //setup CANopen Node
-    /*
-    CO_NODE_SPEC canSpec = {
-        .NodeId = .NODE_ID,
-        .Baudrate = IO::CAN::DEFAULT_BAUD,
-        .Dict = .getObjectDictionary(), Add call to RampupBoard instance
-        .DictLen = .getNumElements(),
-        .EmcyCode = NULL,
-        .TmrMem = appTmrMem,
-        .TmrNum = 16,
-        .TmrFreq = 100,
-        .Drv = &canStackDriver,
-        .SdoBuf = reinterpret_cast<uint8_t*>(&sdoBuffer[0]),
-    };
-
-    CO_NODE canNode;
-
-    CONodeInit(&canNode, &canSpec);
-    CONodeStart(&canNode);
-    CONmtSetMode(&canNode.Nmt, CO_OPERATIONAL);
-    */
+    // Make CAN node
 
+    // Initialize all the CANOpen drivers.
```

Comment:
When I made this, I made it so that the comments were exactly the same as the comments in the CANopen sample, so it would be easy to follow along. I would recommend updating to match the new CANopen sample, so new people still have this advantage